responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -649,4 +676,24 @@ def load(path):
             managers.append(manager)
 
         inst.__dict__['_' + _MANAGERS] = managers
+
+    @staticmethod
+    def load(path):
+        """Load the RAIInsights from the given path.
+
+        :param path: The directory path to load the RAIInsights from.
+        :type path: str
+        :return: The RAIInsights object after loading.
+        :rtype: RAIInsights
+        """
+        # create the RAIInsights without any properties using the __new__
+        # function, similar to pickle
+        inst = RAIInsights.__new__(RAIInsights)
+
+        # load current state
+        RAIInsights._load_data(inst, path)
```

Comment:
maybe at some point we should even move these load methods to a separate file or file(s), since this rai_insights file is already very large

Reply:
Sure. We might need to think about the interfaces for these methods to be exported to another file.